src/extract/transpile/svelte-wrap.js

```diff
@@ -0,0 +1,46 @@
+const tryRequire = require("semver-try-require");
+const _get = require("lodash/get");
+
+const svelte = tryRequire(
+  "svelte",
+  require("../../../package.json").supportedTranspilers.svelte
+);
+
+// eslint-disable-next-line import/order, node/no-unpublished-require
+const svelteCompile = svelte ? require("svelte/compiler") : false;
+
+module.exports = (pTsWrapper) => ({
+  isAvailable: () => svelteCompile !== false,
+  transpile: async (pSource, pTranspileOptions = {}) => {
+    const optionallyCompileTsToJsInSvelte = await svelteCompile.preprocess(
+      pSource,
+      {
+        script: ({ content, attributes }) => {
+          if (attributes.lang !== "ts" || !pTsWrapper.isAvailable()) {
+            return { code: content };
+          }
+          const compiledToTsFromJs = pTsWrapper.transpile(content, {
+            ...pTranspileOptions,
+            tsConfig: {
+              ..._get(pTranspileOptions, "tsConfig", {}),
+              options: {
+                ..._get(pTranspileOptions, "tsConfig.options", {}),
+                importsNotUsedAsValues: "error",
+                jsx: "preserve",
+              },
+            },
+          });
+          return { code: compiledToTsFromJs };
+        },
+      }
+    );
+    const compiledSvelteCode = svelteCompile.compile(
+      optionallyCompileTsToJsInSvelte.code
+    );
+    // remove `import {...} from "svelte/internal";`
+    return compiledSvelteCode.js.code.replace(
```

Comment:
This code removes a real dependency from the output. I think it'd be better to return the `.js.code` as-is. If the end user doesn't want to see dependencies to `svelte/internal` she can always exclude them herself (either in config or in a command line switch).

Reply:
ack